src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
```

Comment:
https://github.com/mate-academy/js_herbivores_and_carnivores/issues/952#issuecomment-1219513446
You can add static property as in issue comment to avoid test fails

```suggestion
```